### DISCUSSION TITLE
关于 feature 和进度

### DISCUSSION THREAD

**Discussion by nerd010** [guopenghui/obsidian-language-learner#58] [General]
hi，要不要考虑放出几个 feature？社区应该会有人接，这样进度会更快一点。

**Comment by guopenghui** [comment]
放出featrue是什么意思? 是列一个开发计划吗

**Comment by nerd010** [comment]
是的

**Comment by nerd010** [comment]
如果有小版本的更新或 bug 的修改也可以先发一个版本，比如，阅读模式下字体大小的调整，多个词典供用户选择等。

很喜欢这个插件，非常有用。感谢开源出来。

**Comment by guopenghui** [comment]
先转到discussion吧
